setup.py

```diff
@@ -50,8 +50,7 @@ def readme():
       license='MIT',
       packages=setuptools.find_packages(exclude=["tests.*", "tests"]),
       install_requires=[
-          "keras>=2.0.0",
-          "opencv-python>=4.1.0"
```

Comment:
I believe this to be a breaking change for the package.
When a package directly uses a dependency, it cannot optimistically rely on the dependency to exist on the system.

Take our example:
We have an external way we generate images (through ffmpeg), transform them to numpy and we pipe them to mtcnn directly. 
At no point in our repo we use OpenCV so we don't require it, but since mtcnn requires it, OpenCV gets installed.

If the requirement is no longer present, this will introduce a silent bug, where our testing system will be fine (as it already has opencv installed), our docker builds will be fine, but at runtime when the code is hit, it will break the production systems.

Reply:
Well, what you are effectively saying here is that if someone installs OpenCV with CMake options to enable python bindings, it is not good enough, even though it is. If CMake detects python when you install OpenCV, it will just build the bindings, and `import cv2` works just fine, without needing to install `opencv-python` through pip.

Reply:
Yes, I fully understand the extent of your problem.
What I'm trying to convey is that this change will fix your case, but at the expense of braking many other existing deployed implementations.

Reply:
I believe this might be of help to your current situation https://stackoverflow.com/questions/39403002/manually-set-package-as-installed-in-python-pip

Reply:
Ok, thanks for the pointer, I can work with that.